package.json

```diff
@@ -8,7 +8,7 @@
     "build": "npm run clean && npm run bundle",
     "#prepare": "yarn clean",
     "clean": "shx rm -rf ./coverage && shx rm -rf ./dist",
-    "prepublish": "yarn clean && yarn code-fix && yarn build",
+    "prepublish": "yarn clean && yarn build",
```

Comment:
why you remove code-fix? 
it's a ESLint, btw or it's generating errors?

Reply:
`code-fix` now exiting with an error code in the terminal. For build sake, I removed this command, but this is actually mistaken. The `standard` library does not have `--silent` terminal flag to prevent any NPM error exit code

Reply:
ok, i agree(understand). but as you change `code-fix` to silent mode, why not to add it back?